core-bundle/src/DependencyInjection/Compiler/SearchIndexerPass.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\PriorityTaggedServiceTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class SearchIndexerPass implements CompilerPassInterface
+{
+    use PriorityTaggedServiceTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container): void
+    {
+        if (!$container->has('contao.search.indexer.delegating')) {
```

Comment:
Isn't it confusing that the service name is `contao.search.indexer.delegating` but the default indexer's name is `contao.search.default_indexer`? Should it not be `contao.search.delegating_indexer`?

Reply:
Ah, that's a leftover anyway. Well spotted :) Fixed in ba3f3b119c6f3fe304e2799c858421c162658539.